temporal-sdk/src/test/java/io/temporal/worker/StickyWorkerTest.java

```diff
@@ -471,74 +462,94 @@ public void workflowsCanBeQueriedAfterEviction() throws Exception {
 
     // Assert
     assertEquals(workflow.getProgress(), GreetingSignalWorkflow.Status.WAITING_FOR_NAME);
+    SDKTestWorkflowRule.assertNoHistoryEvent(
+        wrapper.testEnv.getWorkflowExecutionHistory(execution).getHistory(),
+        EventType.EVENT_TYPE_WORKFLOW_TASK_FAILED);
 
+    wrapper.close();
+  }
+
+  @Test
+  public void workflowCanContinueExecutionAfterBeingEvictedAndQueried() throws Exception {
```

Comment:
Can you confirm that this test would fail if this fix wasn't in place?

Reply:
Yep, I checked it.